unittests/rbusConsumer.cpp

```diff
@@ -50,10 +50,11 @@ static int exec_rbus_get_test(rbusHandle_t handle, const char *param)
 {
   int rc = RBUS_ERROR_BUS_ERROR;
   rbusValue_t val = NULL;
+  rbusValue_t retVal = NULL;
   rbusValueType_t type = RBUS_NONE;
 
   isElementPresent(handle, param);
-  rbusValue_Init(&val);
+  retVal = rbusValue_Init(&val);
```

Comment:
remove the rbusValue_Init as it is not needed for rbus_get()

Reply:
Removed rbusValue_Init as suggested. Updated changes as part of commit 5e0e606f45fa05b41f636a964f3c30819cab16f6